huawei_lte_api/api/Pin.py

```diff
@@ -14,8 +14,8 @@ def simlock(self) -> GetResponseType:
     def save_pin(self) -> GetResponseType:
         return self._session.get('pin/save-pin')
 
-    def operate(self, operate_type: int = 0, current_pin: Optional[int] = None,
-                new_pin: Optional[int] = None, puk_code: Optional[int] = None) \
+    def operate(self, operate_type: str = 0, current_pin: Optional[str] = None,
```

Comment:
Default arg. for operate_type should be '0' (string) not 0 (int).

Reply:
Dont know how I missed that :(

Reply:
Fixed